docs/topic-guides/system-concepts/scheduling.txt

```diff
@@ -6,88 +6,111 @@
+   experiments, GPU notebooks). The fair-share scheduler schedules
+   zero-slot tasks on a FIFO basis. The priority scheduler schedules
+   zero-slot tasks based on priority.
 
 ************
  Fair-Share
 ************
 
-The master allocates cluster resources (slots) among the active
-experiments using a fair-share scheduling policy. In other words, slots
-are divided among the active experiments according to the *demand*
-(number of desired concurrent tasks) of each experiment. For instance,
-in an eight-GPU cluster running two experiments with demands of ten and
+The master allocates cluster resources (*slots*) among the active
+experiments using a weighted fair-share scheduling policy. Slots are
+divided among the active experiments according to the *demand* (number
+of desired concurrent tasks) of each experiment. For instance, in an
+eight-GPU cluster running two experiments with demands of ten and
 thirty, the scheduler assigns two slots and six slots respectively. As
 new experiments become active or the resource demand of an active
 experiment changes, the scheduler will adjust how slots are allocated to
 experiments as appropriate.
 
+The behavior of the fair-share scheduler can be modified by setting the
+*weight* of a workload. A workload's demand for slots is multipled by
+the workload's weight for scheduling purposes; hence, a workload with a
+higher weight will be assigned proportionally more resources than a
+workload with lower weight. The default weight is ``1``. For example, in
+the scenario above, if the weight of the first experiment is set to 3
+and the weight of the second experiment is set to 1, each experiment
+will be assigned four slots.
+
 **********
  Priority
 **********
 
-The master allocates cluster resources (slots) to active tasks based on
-their priority. While tasks of higher priority (lower priority number)
-are pending, no lower priority tasks will be scheduled. For instance, if
-tasks with priorities of five and forty-two are pending, the latter will
-not be scheduled until the former has been. Tasks of equal priority are
-scheduled in the order in which they were created.
-
-By default the priority scheduler will not perform any preemption. If
-preemption is enabled (:ref:`master-configuration`), in scenarios where
-a higher priority task is pending and cannot be scheduled, the scheduler
-will attempt to schedule it by preempting lower priority tasks, starting
-with the lowest priorities.
+The master allocates cluster resources (*slots*) to active tasks based
+on their *priority*. While tasks of higher priority (lower priority
+number) are pending, no lower priority tasks will be scheduled. For
+instance, if tasks with priorities of five and forty-two are pending,
+the latter will not be scheduled until the former has been. Tasks of
+equal priority are scheduled in the order in which they were created.
+
+By default, the priority scheduler does not use preemption. If
```

Comment:
Does it make sense to include a snippet about why preemption is disabled by default? Maybe explain the cost of preempting may be large for certain trials.